src/test/javascript/spec/component/lti/lti13-deep-linking.component.spec.ts

```diff
@@ -76,6 +77,16 @@ describe('Lti13DeepLinkingComponent', () => {
         expect(component.exercises).toContainAllValues(course.exercises!);
     }));
 
+    it('should alert user when no exercise is selected', () => {
+        component.selectedExercises = undefined;
+        component.sendDeepLinkRequest();
+        expect(alertServiceMock.error).toHaveBeenCalledWith('artemisApp.lti13.deepLinking.selectToLink');
+
+        component.selectedExercises = new Set();
+        component.sendDeepLinkRequest();
+        expect(alertServiceMock.error).toHaveBeenCalledTimes(2);
```

Comment:
Nitpick: The second call should be with the same translation key - we should check that here too.
(You could use `toHaveBeenNthCalledWith` for this)